share/site/duckduckgo/base.tx

```diff
@@ -9,7 +9,7 @@
   <head>
     <: include "head.tx" :>
   </head>
-  <body id="pg-<: $f.filebase :>" class="page-<: $f.filebase :><: if $homepage { :> body--home<: } :>">
```

Comment:
@jagtalon @MariagraziaAlastra Why did the `beta` class get added to the homepage in production?!